cmd/promcord/service.go

```diff
@@ -77,6 +79,14 @@ var (
 		TagKeys:     []tag.Key{Guild, Channel, User},
 		Aggregation: view.LastValue(),
 	}
+	// MemberCountView .
+	MemberCountView = &view.View{
+		Name:        "member/count",
+		Measure:     MemberCount,
+		Description: "The number of members",
+		TagKeys:     []tag.Key{Guild, User},
```

Comment:
What is the user TagKey for here?

Reply:
ah, I see... does this log the join event then?
I think there should be a different metric (joins and leaves) aside from count which holds this data as member/count is not user specific so a changing tag here will cause counterintuitive data.